csl-types.rnc

```diff
@@ -10,12 +10,15 @@ div {
     | "book"
     | "broadcast"
     | "chapter"
+      "classic"
+      "collection"
```

Comment:
@bwiernik, these new entries all need to be preceded by a pipe ("|"), which IIRC means OR in RNC.

(we should have some CI to make sure the schema is valid)

Reply:
Tip - easiest way to validate RNC is to use Trang to convert it to RNG. If it converts without error, it's valid.

Reply:
> (we should have some CI to make sure the schema is valid)

Do we have a ticket for this?

Reply:
Just added #197